lib/path.js

```diff
@@ -1053,29 +1053,18 @@ const posix = {
     if (from === to)
       return '';
 
+    // Trim any leading backslashes
```

Comment:
isn't this trimming leading forward slashes?

Reply:
Good catch. Fixed. Seems like the comment has always been wrong and no one noticed it so far.